lib/content-services/src/lib/content-metadata/components/content-metadata-card/content-metadata-card.component.html

```diff
@@ -23,24 +23,6 @@
                 data-automation-id="meta-data-card-edit-aspect">
                 <mat-icon>menu</mat-icon>
             </button>
-            <button *ngIf="!readOnly && hasAllowableOperations()"
-                mat-icon-button
-                (click)="toggleEdit()"
-                [attr.title]="'CORE.METADATA.ACTIONS.EDIT' | translate"
```

Comment:
check if these removed translations are not used anywhere, and cleanup the `en.json` file